LayoutGrid/LayoutGrid.jsx

```diff
@@ -0,0 +1,78 @@
+import { h } from "preact";
+import MaterialComponent from "../MaterialComponent";
+
+class LayoutGrid extends MaterialComponent {
+	constructor() {
+		super();
+		this.componentName = "layout-grid";
+	}
+	materialDom(props) {
+		return (
+			<div className="mdc-layout-grid" ref={control=>{this.control = control;}} {...props}>
```

Comment:
this className is not needed, every component's root element will be given a className of 'mdc' + componentName.
e.g.
https://github.com/prateekbh/preact-material-components/blob/master/Button/Button.jsx#L23-L28

source: https://github.com/prateekbh/preact-material-components/blob/master/MaterialComponent.js#L30